app/scripts/lib/setupSentry.js

```diff
@@ -28,72 +29,269 @@ export const ERROR_URL_ALLOWLIST = {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    previousUserTraits: false,
+    segmentApiCalls: false,
+    traits: false,
   },
   NetworkController: {
+    networkConfigurations: false,
     networkId: true,
-    networkStatus: true,
+    networksMetadata: true,
     providerConfig: {
+      chainId: false,
+      id: true,
       nickname: true,
+      rpcPrefs: false,
+      rpcUrl: false,
       ticker: true,
       type: true,
     },
+    selectedNetworkClientId: false,
+  },
+  NftController: {
+    allNftContracts: false,
+    allNfts: false,
+    ignoredNfts: false,
+  },
+  NotificationController: {
+    notifications: false,
   },
   OnboardingController: {
     completedOnboarding: true,
     firstTimeFlowType: true,
+    onboardingTabs: false,
     seedPhraseBackedUp: true,
   },
+  PermissionController: {
+    subjects: false,
+  },
+  PermissionLogController: {
+    permissionActivityLog: false,
+    permissionHistory: false,
+  },
+  PhishingController: {},
   PreferencesController: {
+    advancedGasFee: true,
     currentLocale: true,
+    disabledRpcMethodPreferences: true,
+    dismissSeedBackUpReminder: true,
     featureFlags: true,
     forgottenPassword: true,
-    ipfsGateway: true,
-    preferences: true,
+    identities: false,
+    incomingTransactionsPreferences: true,
+    infuraBlocked: true,
+    ipfsGateway: false,
+    isLineaMainnetReleased: true,
+    knownMethodData: false,
+    ledgerTransportType: true,
+    lostIdentities: false,
+    openSeaEnabled: true,
+    preferences: {
+      autoLockTimeLimit: true,
+      hideZeroBalanceTokens: true,
+      showFiatInTestnets: true,
+      showTestNetworks: true,
+      useNativeCurrencyAsPrimaryCurrency: true,
+    },
+    selectedAddress: false,
+    snapRegistryList: true,
+    theme: true,
+    transactionSecurityCheckEnabled: true,
+    use4ByteResolution: true,
+    useAddressBarEnsResolution: true,
     useBlockie: true,
+    useCurrencyRateCheck: true,
+    useMultiAccountBalanceChecker: true,
+    useNftDetection: true,
     useNonceField: true,
     usePhishDetect: true,
+    useTokenDetection: true,
   },
   SignatureController: {
     unapprovedMsgCount: true,
+    unapprovedMsgs: false,
     unapprovedPersonalMsgCount: true,
+    unapprovedPersonalMsgs: false,
+    unapprovedTypedMessages: false,
     unapprovedTypedMessagesCount: true,
   },
+  SmartTransactionsController: {
+    smartTransactionsState: {
+      fees: {
+        approvalTxFees: true,
+        tradeTxFees: true,
+      },
+      liveness: true,
+      smartTransactions: false,
+      userOptIn: true,
+    },
+  },
+  SnapController: {
+    snapErrors: false,
+    snapStates: false,
+    snaps: false,
+  },
+  SnapsRegistry: {
+    database: false,
+    lastUpdated: false,
+  },
+  SubjectMetadataController: {
+    subjectMetadata: false,
+  },
+  SwapsController: {
+    swapsState: {
+      approveTxId: false,
+      customApproveTxData: true,
```

Comment:
This data likely just contains a custom approval limit, but I'll exclude it for now because I'm not 100% certain

```suggestion
      customApproveTxData: false,
```